src/app/components/link-custom/link-custom.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames/bind';
+import { Link } from 'react-router-dom';
+import { ArrowIcon, TextLinkIcon } from '@/icons';
+
+import styles from './link-custom.module.scss';
+
+type LinkCustomProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+> & {
+  href: string;
+  icon?: boolean;
+  button?: boolean;
+  size?: 'large' | 'medium' | 'small';
+  outlined?: boolean;
+  rounded?: boolean;
+  invertIconColor?: boolean;
+  className?: string;
+};
+
+export const LinkCustom = ({
+  children,
+  href,
+  icon = true,
+  button = false,
+  size = 'large',
+  outlined = false,
+  rounded = false,
+  invertIconColor = false,
+  className = '',
+  ...props
```

Comment:
In my opinion it's little bit confusing that every modificator such as `button`, `outlined` and etc. is `false` by defalut, but icon is not, probably making all the same way will be better idea.

I think when i using component i want to turn on modifications using props instead of turning off. For example if i need an icon i will explicitly pass prop meaning that i need an icon, if i don't need it i just ignore this prop

Reply:
Removed 'icon' prop